createtable_test.go

```diff
@@ -86,3 +89,17 @@ func ExampleCreateTableBuilder_NumDefine() {
 	// Output:
 	// 5
 }
+
+func TestCreateTableGetFlavor(t *testing.T) {
+	a := assert.New(t)
+	ctb := newCreateTableBuilder()
+	postgresFlavor := PostgreSQL
```

Comment:
It's not necessary to assign a const to a local variable. Doing this will make the test code less readable. It's recommended to use const in assertions directly.